ucrop/src/main/java/com/yalantis/ucrop/util/BitmapLoadUtils.java

```diff
@@ -24,6 +25,7 @@
     private static final String TAG = "BitmapLoadUtils";
 
     @Nullable
+    @TargetApi(Build.VERSION_CODES.JELLY_BEAN)
```

Comment:
Had to add this to get lint to ignore the `ParcelFileDescriptor` -> `Closeable` cast.